dr_evaluation/get_greenbutton_id.py

```diff
@@ -0,0 +1,38 @@
+import dataclient
+m = dataclient.MDALClient("corbusier.cs.berkeley.edu:8088")
```

Comment:
Can you replace this call with its equivalent in Mortar? The MDAL service won't be up forever